share/qt/Info.plist.in

```diff
@@ -30,6 +30,12 @@
 
   <key>CFBundleExecutable</key>
   <string>Bitcoin-Qt</string>
+  
+  <key>CFBundleName</key>
+  <string>Bitcoin-Qt</string>
```

Comment:
For the records: this is required so that the OS can detect a manual rename of the application. If current (manually renamed) application name is != `CFBundleName`, it won't use localization over `Base.lproj` to rename the application display name.